Tasks/Inc/Task_RTOSBPS.h

```diff
@@ -0,0 +1,54 @@
+/* Copyright (c) 2022 UT Longhorn Racing Solar */
+
+//need to add the proper #include files 
+
+#ifndef Task_RTOSBPS_H
+#define Task_RTOSBPS_H
+
+//#include
+
+
+/**
+ * @brief   
+ * @param   
+ * @return  
+ */
+void RTOS_SemPend(void);
```

Comment:
Could you rename to RTOS_BPS_blah. I want something that no other RTOS function would have in their name so I would like to include BPS to prevent compiler errors for future porting.

Reply:
👍🏼